test/RightsService.Business.UnitTests/Commands/Role/GetRoleCommandTests.cs

```diff
@@ -1,43 +1,269 @@
         public void OneTimeSetUp()
         {
-            _dbRole = new DbRole {
-                Id = Guid.NewGuid()
+            _mocker = new AutoMocker();
+            _operationResultGetUsersData = new Mock<Response<IOperationResult<IGetUsersDataResponse>>>();
+
+            _validRoleId = Guid.NewGuid();
+
+            _dbUser = new DbUser() { UserId = Guid.NewGuid() };
+            _dbRight = new DbRight() { Id = 1, Name = "rightName", Description = "rightDescription" };
+            _dbRole = new DbRole()
+            {
+                Id = _validRoleId,
+                Name = "roleName",
+                Description = "roleDescr",
+                CreatedBy = Guid.NewGuid(),
+                CreatedAt = DateTime.UtcNow,
+                IsActive = true,
+                Rights = new List<DbRoleRight>() { new DbRoleRight() { Right = _dbRight } },
+                Users = new List<DbUser>() { _dbUser }
+            };
+
+            _userData = new UserData(_dbUser.UserId, Guid.NewGuid(), "fName", "mName", "lName", "s", 1.0f, true);
+            _userInfo = new UserInfo()
+            {
+                Id = _userData.Id,
+                FirstName = _userData.FirstName,
+                LastName = _userData.LastName,
+                MiddleName = _userData.MiddleName
+            };
+            _roleInfo = new RoleInfo
+            {
+                Id = _dbRole.Id,
+                Name = _dbRole.Name,
+                Description = _dbRole.Description,
+                CreatedAt = _dbRole.CreatedAt,
+                CreatedBy = _dbRole.CreatedBy,
+                Rights = new List<RightResponse>() { _rightResponse },
+                Users = new List<UserInfo>() { _userInfo }
             };
 
-            _roleInfo = new UserInfo
+            _rightResponse = new RightResponse()
             {
-                Id = _dbRole.Id
+                Id = _dbRight.Id,
+                Name = _dbRight.Name,
+                Description = _dbRight.Description
             };
 
+            _roleResponse = new RoleResponse()
+            {
+                Role = _roleInfo,
+                Errors = new List<string>()
+            };
+
+            _command = new GetRoleCommand(
+                _mocker.GetMock<ILogger<GetRoleCommand>>().Object,
+                _mocker.GetMock<IRoleRepository>().Object,
+                _mocker.GetMock<IRightRepository>().Object,
+                _mocker.GetMock<IRoleInfoMapper>().Object,
+                _mocker.GetMock<IUserInfoMapper>().Object,
+                _mocker.GetMock<IRightResponseMapper>().Object,
+                _mocker.GetMock<IRequestClient<IGetUsersDataRequest>>().Object
+            );
         }
 
         [SetUp]
         public void SetUp()
         {
-            _mocker = new AutoMocker();
-            _command = _mocker.CreateInstance<IGetRoleCommand>();
+            _operationResultGetUsersData.Reset();
+            _mocker.GetMock<ILogger<GetRoleCommand>>();
```

Comment:
maybe you want here do reset, or for what this is

Reply:
++